stack/indexer/deb/debian/preinst

```diff
@@ -60,6 +61,14 @@ case "$1" in
     ;;
 
     upgrade)
+        # Move the securityconfig files if they exist (4.3.x versions)
```

Comment:
Shouldn't this be done on uninstall better? Can those files be changed?

Reply:
As I commented in the issue, this process follows: `prerm`, new `preinst`, old `postrm`, new `postinst`. So, in an upgrade, is necessary to move these files on the `preinst` stage, and the uninstallation of the package does not execute this stage.

Reply:
Please link the documentation.